tools/quake3/common/polylib.c

```diff
@@ -461,15 +461,15 @@ winding_t *BaseWindingForPlane( vec3_t normal, vec_t dist ){
    ==================
  */
 winding_t   *CopyWinding( winding_t *w ){
-	int size;
+	size_t size;
 	winding_t   *c;
 
 	if ( !w ) {
 		Error( "CopyWinding: winding is NULL" );
 	}
 
 	c = AllocWinding( w->numpoints );
-	size = sizeof( *w ) + sizeof( *w->p ) * w->numpoints;
+	size = (size_t)sizeof( *w ) + sizeof( *w->p ) * w->numpoints;
```

Comment:
This seems a bit pointless. The type of a `sizeof` expression is already `size_t`.

Reply:
thanks, I will report it on netradiant side too (the point is also to have the same thing on both side, but it must makes sense yes)